RELEASE_NOTES.md

```diff
@@ -32,56 +31,47 @@ CheckoutConfiguration(
     }
 }
 ```
-- Address Lookup functionality for Card Component. 
-  - You can enable this feature by setting your address configuration to lookup while building your card configuration as follows:
-    ```kotlin
-    CheckoutConfiguration(environment = environment, clientKey = clientKey) {
-        card {
-            setAddressConfiguration(AddressConfiguration.Lookup())
-        }
-    }
-    ```
-  - If you're integrating with Drop-in:
-    - Implement the mandatory `onAddressLookupQueryChanged(query: String)` callback and optional `onAddressLookupCompletion(lookupAddress: LookupAddress)` callback.
-    - Pass the result of these actions by using `AddressLookupDropInServiceResult` class.
-  - If you're integrating with standalone `CardComponent`:
-    - Set `AddressLookupCallback` via `CardComponent.setAddressLookupCallback(AddressLookupCallback)` function to receive the related events.
-    - Pass the result of these actions by calling `CardComponent.setAddressLookupResult(addressLookupResult: AddressLookupResult)`.
-    - Delegate back pressed event to `CardComponent` by calling `CardComponent.handleBackPress()` which returns true if the back press is handled by Adyen SDK and false otherwise.
-- Add support for Multibanco voucher.
-- Permission request is now being delegated to the `ActionComponentCallback`, `SessionComponentCallback` or `ComponentCallback` to handle it and return result through callback.
-  - If not handled, a toast will be shown stating that permission is not granted.
-- For voucher actions which have no `url` or `downloadUrl`, "Save as image" option will be offered to save the Voucher in `Downloads` folder.
-  - Vouchers will save an image to user's phone with the following name format "Payment method type" + "Formatted data and time" (e.g. multibanco-2024-01-09T16_41_10).
-- Set your own `AdyenLogger` instance with `AdyenLogger.setLogger`. This gives the ability to intercept logs and handle them in your own way.
+
+- For the Card Component, you can use the new [Address Lookup functionality](docs/ADDRESS_LOOKUP.md). 
+- For voucher actions: when the `url` or `downloadUrl` is not included, the shopper has the option to select **Save as image** and save the voucher to the device's `Downloads` folder.
+- You can now set your own `AdyenLogger` instance with `AdyenLogger.setLogger`. This gives the ability to intercept logs and handle them in your own way.
 - Payment methods:
+  - Multibanco. Payment method type: **multibanco**.
   - Pay Easy. Payment method type: **econtext_atm**.
   - Convenience Stores Japan. Payment method type: **econtext_stores**
   - Online Banking Japan. Payment method type: **econtext_online**.
   - Seven-Eleven: Payment method type: **econtext_seven_eleven**
 
```

Comment:
Can we add a point here that we added a readme for setting up the testing app and to migrate from the old one they can use the "How to migrate" section in https://github.com/Adyen/adyen-android/pull/1505

Reply:
added.